app/styleguide/core/typography/typography.md

```diff
@@ -56,18 +56,12 @@ Available font-weight variables
 
 ### Block elements
 
-**H1** <html><h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua</h1></html>
+**H1** <html><span style="font-size: 18px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua</span></html>
```

Comment:
- [ ] Don't use `<html>` here because it's invalid

Use ` ```<h1>Foo</h1>``` ` instead, which should render a valid h1 tag.

Reply:
I tried it with ``` but id didn't work. I just put <h1>Foo</h1> there and it renders a h1 tag. However, it also adds some classes to h1 which override the global styles... Not sure how to proceed here.

Reply:
Done. I noticed that _reboot.scss adds a bottom margin to h and p tags. Is that ok?